cmd/ocm-backplane/accessrequest/createAccessRequest.go

```diff
@@ -0,0 +1,253 @@
+package accessrequest
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/andygrunwald/go-jira"
+	ocmcli "github.com/openshift-online/ocm-cli/pkg/ocm"
+	"github.com/openshift/backplane-cli/pkg/elevate"
+	"github.com/openshift/backplane-cli/pkg/ocm"
+	"github.com/openshift/backplane-cli/pkg/utils"
+	logger "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+const (
+	ProductionOCMUrl = "https://api.openshift.com"
+)
+
+var (
+	options struct {
+		reason              string
+		notificationIssueID string
+		pendingDuration     time.Duration
+		approvalDuration    time.Duration
+	}
+)
+
+// newCreateAccessRequestCmd returns cobra command
+func newCreateAccessRequestCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:           "create",
+		Short:         "Creates a new pending access request",
+		Args:          cobra.ExactArgs(0),
+		SilenceUsage:  true,
+		SilenceErrors: true,
+		RunE:          runCreateAccessRequest,
+	}
+
+	cmd.Flags().StringVarP(
+		&options.reason,
+		"reason",
+		"r",
+		"",
+		"Reason/justification passed through the access request to the customer. "+
+			"Reason will be read from the kube context (unless --cluster-id is set) or prompted if the option is not set.")
+
+	cmd.Flags().StringVarP(
+		&options.notificationIssueID,
+		"notification-issue",
+		"n",
+		"",
+		"JIRA issue used for notifications when the access request is approved or denied. "+
+			"Issue needs to belong to the OHSS project on production and to the SDAINT project for staging & integration. "+
+			"Issue will automatically be created in the proper project if the option is not set.")
+
+	cmd.Flags().DurationVarP(
+		&options.approvalDuration,
+		"approval-duration",
+		"d",
+		8*time.Hour,
+		"The maximal period of time during which the access request can stay approved")
+
+	return cmd
+}
+
+// runCreateAccessRequest creates access request for the given cluster
+func runCreateAccessRequest(cmd *cobra.Command, args []string) error {
+	clusterID, err := getClusterID(cmd)
+	if err != nil {
+		return fmt.Errorf("failed to compute cluster ID: %v", err)
+	}
+
+	ocmConnection, err := ocmcli.NewConnection().Build()
+	if err != nil {
+		return fmt.Errorf("failed to create OCM connection: %v", err)
+	}
+	defer ocmConnection.Close()
+
+	accessRequest, err := getAccessRequest(clusterID, ocmConnection)
+
+	if err != nil {
+		return err
+	}
+
```

Comment:
You may not need to build ocm connection here, it may be part of ocm interface implementation 
ref : https://github.com/openshift/backplane-cli/blob/4ce37a3a4a21e8b11f16d1ec69d9103239d0fe12/pkg/ocm/ocmWrapper.go#L61

Reply:
As said in the "import `github.com/openshift-online/ocm-cli/pkg/ocm` package" comment:
- The `create` subcommand may need to perform several calls in the row to OCM, hence it is probably best to not re-create the OCM client between the calls. Right?
- Hence the OCM wrapper methods interacting with the access transparency sub-API of OCM need to take the OCM client as parameter.
- Last but not least: some other methods of the OCM wrapper already take the OCM client as parameter, probably for the same reason: repetitive calls to the OCM API.

Reply:
Yeah, I understand your points about the OCM wrapper. We could probably refactor the OCM wrapper to cache the connection and reuse it in subsequent calls. I'm not too sure if initiating a connection in the command runner is a good pattern or an anti-pattern.

Reply:
I think it is ok a long as we add some `Close()` method